src/server/auth/server/api_server.go

```diff
@@ -301,7 +301,7 @@ func (a *apiServer) getClusterRoleBinding(ctx context.Context) (*auth.RoleBindin
 			return nil, errors.New("cached cluster binding had unexpected type")
 		}
 
-		if bindings.Entries == nil {
+		if len(bindings.Entries) == 0 {
```

Comment:
why is this necessary?

Reply:
to catch when `bindings.Entries` isn't nil but an empty map.
https://go.dev/play/p/_igsFVhZJGt